gpu-driver-util/main.go

```diff
@@ -177,12 +181,14 @@ func resolveKernelModuleType(gpuDevices []string, searchMap map[string]GPUDevice
 	if requiresOpenRM && requiresProprietary {
 		return "", fmt.Errorf("unsupported GPU topology")
 	} else if requiresOpenRM {
-		return KernelModuleTypeOpen, nil
+		kernelModuleType = KernelModuleTypeOpen
 	} else if requiresProprietary {
+		kernelModuleType = KernelModuleTypeProprietary
```

Comment:
Did you mean to remove the `return` statement after this line?

Reply:
ah yes, good catch. Will remove it

Reply:
Please check now